### PR TITLE
Created preact component for crayons dropdown

### PR DESCRIPTION
**Pull request by nickytonline**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature (Part of subscription surfacing work)
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Creates the `<Dropdown />` component and accompanying Storybook stories. Available via `import { Dropdown } from '@crayons';`

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)

dropdown

## Added tests?

- [x] yes
- [ ] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

MMA fighter dropping like a fly in the octagon

### CODE REVIEW

**Comment by nickytonline** [comment]
> Not sure if this is part of this PR, but I feel like this could show up on click actually. Thoughts? Maybe we could have a component setting whether or not dropdown should show up on click or on hover..?

The button is just to show the dropdown, i.e. not part of the component. This is more an implementation detail depending on how you want to use the `<Dropdown />` component within a component.

File: jest.config.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ module.exports = {
   ],
   moduleNameMapper: {
     '\\.(svg|png)$': '<rootDir>/empty-module.js',
+    '^@crayons(.*)$': '<rootDir>/app/javascript/crayons$1',
```
Comment (nickytonline) [inline jest.config.js]: Allows us to use the webpack alias `@acrayons` in jest as well.

File: app/javascript/crayons/Dropdown/__stories__/dropdown-css-helper.scss [hunk 2]
```diff
@@ -0,0 +1,7 @@
+.dropdown-trigger-container {
```
Comment (nickytonline) [inline app/javascript/crayons/Dropdown/__stories__/dropdown-css-helper.scss]: This CSS is only used for the `<Dropdown />` Storybook stories.

File: jest.config.js (hunk 1 above)
Comment (ludwiczakpawel) [reply jest.config.js]: what `jest` stands for? (in file name)... just out of curiosity..

Comment (nickytonline) [reply jest.config.js]: This is the configuration file for the testing framework we use in the frontend. See https://jestjs.io

**Pull request merged**